src/Components/Posts.js

```diff
@@ -0,0 +1,14 @@
+import Post from "/Users/younestahiri/Documents/8-0 Projects/8-0-react-introduction-lab/src/Components/Post.js";
```

Comment:
You want to make sure to use relative paths because I, as another developer looking at your code, do not have access to "/Users/younestahiri/Documents/" part of the file. 

You want to use the "./" or "../" notation to tell your files where other files are coming from.